package.json

```diff
@@ -87,7 +90,8 @@
     "stylelint-scss": "^3.18.0",
     "testcafe": "^1.8.4",
     "testcafe-angular-selectors": "^0.4.1",
-    "testcafe-reporter-html": "^1.4.6"
+    "testcafe-reporter-html": "^1.4.6",
+    "yn": "^4.0.0"
```

Comment:
Seems not required.

Reply:
You're right! `yn` can be safely removed from the root `package.json` file.

Only used here:
```sh
$ git grep "import yn" -- "*.ts"
packages/manager/tools/webpack-dev-server/src/config.ts:import yn from 'yn';
```